Dockerfile

```diff
@@ -42,4 +42,9 @@ RUN sed -e 's|-bs-no-version-header|-bs-no-version-header", "-warn-error -a+5+6+
 ################################################################################
 FROM quay.io/change-metrics/monocle-exe:latest
 
+# https://hackage.haskell.org/package/HsOpenSSL-x509-system-0.1.0.4/docs/src/OpenSSL.X509.SystemStore.Unix.html#contextLoadSystemCerts
+# Nix cacert installs a link to the nix store in /etc/ssl/certs/ca-bundle.crt
+# Let's copy it in /etc/pki/tls/certs as a regular file
+RUN mkdir -p /etc/pki/tls/certs/ && cp /etc/ssl/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
I think it would be better to simply run that comment in https://github.com/change-metrics/monocle/blob/master/nix/default.nix#L481 . Just make the path relative:

  mkdir -p etc/pki/tls/certs && ln -s /etc/ssl/certs/ca-bundle.crt etc/pki/tls/certs/ca-bundle.crt

Reply:
The `isRegularFile` here https://hackage.haskell.org/package/HsOpenSSL-x509-system-0.1.0.4/docs/src/OpenSSL.X509.SystemStore.Unix.html#contextLoadSystemCerts might cause trouble if we keep the link.

Alright I'll do the change within nix instead of inside the Dockerfile.